app.js

```diff
@@ -0,0 +1,14 @@
+const close = document.querySelector('.close');
+const menuBars = document.querySelector('.menu-bars');
+const menu = document.querySelector('#mobile-nav');
+
+function showMobileMenu() {
+  menu.style.display = 'block';
+}
+
+function hideMobileMenu() {
+  // menu.style.display = 'none';
+}
+
+menuBars.addEventListener('click', showMobileMenu());
+close.addEventListener('click', hideMobileMenu());
```

Comment:
According to the project requirements, the mobile menu should disappear when the user clicks any menu item. In your project, however, the menu does not disappear 🚫. 
- [ ]  Please kindly make the menu disappear when user clicks  any menu item.